packages/terra-dialog-modal/src/DialogModal.jsx

```diff
@@ -0,0 +1,140 @@
+import React from 'react';
+import AbstractModal from 'terra-abstract-modal';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import styles from './DialogModal.scss';
+
+const cx = classNames.bind(styles);
+
+const widthFromSize = {
+  320: 320,
+  480: 480,
+  640: 640,
+  800: 800,
+  960: 960,
+  1120: 1120,
+  1280: 1280,
+  1440: 1440,
+  1600: 1600,
+  1760: 1760,
+  1920: 1920,
+};
+
+const propTypes = {
+ /**
+  * Aria Label of the dialog modal.
+  */
+  ariaLabel: PropTypes.string.isRequired,
+ /**
+  * Header of the dialog modal.
+  */
+  header: PropTypes.element.isRequired,
+ /**
+  * Footer of the dialog modal.
+  */
+  footer: PropTypes.element.isRequired,
+ /**
+  * Contents of the dialog modal.
+  */
+  children: PropTypes.node,
+  /*
+   * Callback function indicating a close condition was met.
+   */
+  onRequestClose: PropTypes.func.isRequired,
+  /*
+   * Toggle to show dialog modal or not.
+   */
+  isOpen: PropTypes.bool,
+  /*
+   * Toggle to focus on dialog modal or not.
+   */
+  isFocused: PropTypes.bool,
+  /**
+   * A callback function to let the containing component (e.g. modal) to regain focus.
+   */
+  releaseFocus: PropTypes.func,
+  /**
+   * A callback function to request focus from the containing component (e.g. modal).
+   */
+  requestFocus: PropTypes.func,
+  /**
+   * Width of the dialog modal. Defaults to 480px.
```

Comment:
We need to list the width options in the descriptions so consumers know what options are available. The proptable renders as follows:
![screen shot 2018-06-11 at 2 24 24 pm](https://user-images.githubusercontent.com/14099737/41252598-34f305ac-6d83-11e8-86c5-55d300adac13.png)